modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,306 @@
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - <example-master-node> <.>
+----
+<.> The amount of storage available to the PV. Adjust this value for your requirements.
+<.> Replace this value with the node to attach this PV to.
++
+[TIP]
+====
+Run the following command to list the available nodes:
+
+[source,terminal]
+----
+$ oc get nodes
+----
+====
+
+.. Verify the creation of the PV by running the following command:
++
+[source,terminal]
+----
+$ oc get pv
+----
++
+.Example output
+[source,terminal]
+----
+NAME                    CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM   STORAGECLASS    REASON   AGE
+etcd-backup-pv-fs       100Gi      RWX            Delete           Available           local-storage            10s
+----
+
+.. Create a PVC named `etcd-backup-pvc.yaml` with contents such as the following:
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: etcd-backup-pvc
+spec:
+  accessModes:
+  - ReadWriteMany
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 10Gi <.>
+  storageClassName: local-storage
+----
+<.> The amount of storage available to the PVC. Adjust this value for your requirements.
+
+.. Apply the PVC by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f etcd-backup-pvc.yaml
+----
+
+. Create a custom resource definition (CRD) file named `etcd-recurring-backups.yaml`. The contents of the created CRD define the schedule and retention type of automated backups.
++
+For the default retention type, use contents such as the following:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Backup
+metadata:
+  name: etcd-recurring-backup
+spec:
+  etcd:
+    schedule: "20 4 * * *" <.>
+    timeZone: "UTC"
+    retentionPolicy:
+      retentionType: "" <.>
+    pvcName: etcd-backup-pvc
+----
+<.> The `CronTab` schedule for recurring backups. Adjust this value for your needs.
+<.> The `""` retention type specifies the default type and size.
++
+To use retention based on the maximum number of backups, add the following key-value pairs to the `retentionPolicy`:
++
+[source,yaml]
+----
+retentionPolicy:
+  retentionType: RetentionNumber
+  retentionNumber:
+    maxNumberOfBackups: 5 <.>
+----
+<.> The maximum number of backups to retain. Adjust this value for your needs.
++
+For retention based on the file size of backups, use the following:
++
+[source,yaml]
+----
+retentionPolicy:
+  retentionType: RetentionSize
+  retentionSize:
+    maxSizeOfBackupsGb: 10 <.>
+----
+<.> The maximum file size of the retained backups in gigabytes. Adjust this value for your needs.
+
+. Create the cronjob defined by the CRD by running the following command:
++
+[source,terminal]
+----
+$ oc create -f etcd-recurring-backup.yaml
+----
+
+. To find the created cronjob, run the following command:
++
+[source,terminal]
+----
+$ oc get cronjob -n openshift-etcd
+----
+
+[IMPORTANT]
```

Comment:
no troubleshooting tips went into the docs? :/ 

https://docs.google.com/document/d/1iIyJYAl2G70RPsPY3wF65i06PGFWBJs-Hk7z68tFPEM/edit#heading=h.cwfnvnd2798b

Reply:
fair enough, hope this doesn't create too many support questions on the forum.